horace_core/sqw/file_io/@faccess_sqw_v3_2/private/upgrade_file_format_.m

```diff
@@ -80,6 +80,19 @@
 end
 exp = Experiment(heads);
 sq = sq.change_header(exp);
+runid = zeros(numel(heads),1);
+for i=1:numel(heads)
+    if iscell(heads)
+        runid(i) = rundata.extract_id_from_filename(heads{i}.filename);
+    else
+        runid(i) = rundata.extract_id_from_filename(heads(i).filename);
+    end
+end
+ids = 1:numel(heads);
+if any(isnan(runid))
+    runid = ids;
+end
```

Comment:
```matlab
if any(isnan(runid))
    runid = 1:numel(heads);
end
```

Reply:
It looks like it is merge bug here. Fixed